rasa/shared/nlu/training_data/formats/wit.py

```diff
@@ -15,22 +16,20 @@ def read_from_json(self, js: Dict[Text, Any], **kwargs: Any):
         """Loads training data stored in the WIT.ai data format."""
         training_examples = []
 
-        for s in js["data"]:
+        for s in js["utterances"]:
             entities = s.get(ENTITIES)
             if entities is None:
                 continue
             text = s.get(TEXT)
-            intents = [e["value"] for e in entities if e["entity"] == INTENT]
-            intent = intents[0].strip('"') if intents else None
-
-            entities = [
-                e
-                for e in entities
-                if ("start" in e and "end" in e and e["entity"] != INTENT)
-            ]
+            intent = s.get("intent", USER_INTENT_OUT_OF_SCOPE)
```

Comment:
why not `""` for the empty intent? does this mean the utterance simply has no `intent` key if the intent was "out of scope"?

Reply:
just found the comment on their API spec website:
> Out-of-scope utterances won't have the intent field set. See negative utterances.

I think this would be useful to add as a comment